website/templates/emails/pending_embargo_termination_admin.html.mako

```diff
@@ -11,11 +11,13 @@
     ${initiated_by} initiated a request to end the embargo for a registration of ${project_name}. The embargoed registration can be viewed here: ${registration_link}.<br>
     % endif
     <br>
-    To approve this change and to make this registration public immediately, click the following link: ${approval_link}.<br>
+    Approve this change and make this registration public immediately: ${approval_link}.<br>
```

Comment:
This should also be an <a> tag wrapping Click Here, as above. Please check through the rest of the document for these.